src/content/en/2022/jamstack.md

```diff
@@ -19,4 +21,424 @@ featured_stat_label_3: TODO
 unedited: true
 ---
 
-## TODO
+## Introduction
+
+One of the biggest problems in writing about Jamstack is defining what, exactly, the Jamstack is. Here are three different definitions (we have emphasized some words):
+
+1. Fast and secure sites and apps delivered by pre-rendering files and serving them directly from a CDN, removing the requirement to manage or run web servers.
+
+2. Jamstack is an **architecture** designed to make the web **faster**, more secure, and easier to scale. It builds on many of the tools and workflows which developers love, and which bring maximum productivity.
+
+3. Jamstack is an **architectural approach** that **decouples** the web experience layer from data and business logic, improving flexibility, scalability, **performance**, and maintainability.
+
+All three of the above definitions come from <a hreflang="en" href="https://jamstack.org/">Jamstack.org</a>: in <a hreflang="en" href="https://web.archive.org/web/20200331214426/https://jamstack.org/">2020</a>, <a hreflang="en" href="https://web.archive.org/web/20210924115533/https://jamstack.org/what-is-jamstack/">2021</a>, and <a hreflang="en" href="https://web.archive.org/web/20220809174445/https://jamstack.org/">2022</a> respectively. It's hard to think of a more authoritative source for the definition of Jamstack, so it's fair to say the definition is something of a moving target.
+
+But as the emphasized words demonstrate, there's clearly some continuity: the sites should be fast, they should be pre-rendered, and they should use an architectural approach that decouples "where you get your data" from "how you render your data". Even if a precise dictionary definition is hard to come by, Jamstack developers know what you mean when you say "Jamstack": you've got a site that loads really quickly, renders a lot of its useful content once, at build time, and retrieves additional data (if it needs to) via JavaScript.
+
+<p class="note">Disclosure: the two authors of this report were Netlify employees. This report and the underlying analysis were reviewed and approved by multiple volunteers not affiliated with Netlify.</p>
+
+## Quantifying the Jamstack: what counts?
+
+But the problem gets more tricky when you're trying to put together the 2022 Web Almanac. When you're dealing with millions of websites, "I know it when I see it" can't be your definition. How do we quantify the Jamstack? How do we precisely identify it so we can learn about it? We started by asking ourselves a series of questions.
+
+### Is every static site a Jamstack site?
+
+That seems like it should be an obvious "yes": if the page is flat HTML that renders all at once then it certainly sounds like it should be Jamstack. But what about all those pages people built in the 90s, before JavaScript was popular and before servers did much? Are they Jamstack? It felt like they weren't, not every static site is a Jamstack site. So we tried to think of why.
```

Comment:
I'm not sure that's what @seldo meant by this? I think he meant Web 1.0 websites that were static, before dynamic, interactive sites came along when servers moved from serving static sites to more dynamic content?

Reply:
Yes that's exactly it.

Reply:
What about this:

```suggestion
That seems like it should be an obvious "yes": if the page is flat HTML that renders all at once then it certainly sounds like it should be Jamstack. But what about all those pages people built in the 90s, before JavaScript was popular and before sites were as dynamic as they are now? Are they Jamstack? It felt like they weren't, not every static site is a Jamstack site. So we tried to think of why.
```

Reply:
Hmm, actually this might complicate things given the large portion of *static sites* the report covers.

How about mentioning the option to choose between whether you're rendering static files or serving dynamic (i.e. personalised) content?

Reply:
The main point I think is about the options we have. Monoliths. Serverless. Edge. Static. Hybrid. All the other stuff in between.

Reply:
Maybe just leave as is then? I think it's understandable.